picard/ui/metadatabox.py

```diff
@@ -427,7 +428,7 @@ def _update_tags(self):
         if not (files or tracks):
             return None
 
-        tag_diff = TagDiff()
+        tag_diff = TagDiff(max_length_diff=config.setting["ignore_track_duration_difference_under"])
```

Comment:
Any reason to initialize the value here instead of just fetching it in the __init__ for TagDiff?

Reply:
> Any reason to initialize the value here instead of just fetching it in the __init__ for TagDiff?

I prefer to not link the class TagDiff to config options directly.